test/vault_cluster_dynamodb_backend_test.go

```diff
@@ -0,0 +1,52 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/random"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+)
+
+const VAULT_CLUSTER_DYNAMODB_BACKEND_PATH = "examples/vault-dynamodb-backend"
+
+const VAR_DYNAMO_TABLE_NAME = "dynamo_table_name"
+
+// Test the Vault with DynamoDB Backend example by:
+//
+// 1. Copy the code in this repo to a temp folder so tests on the Terraform code can run in parallel without the
+//    state files overwriting each other.
+// 2. Build the AMI in the vault-consul-ami example with the given build name
+// 3. Deploy that AMI using the example Terraform code
+// 4. SSH to a Vault node and initialize the Vault cluster
+// 5. SSH to each Vault node and unseal it
+// 6. Connect to the Vault cluster via the ELB
+func runVaultWithDynamoBackendClusterTest(t *testing.T, amiId string, awsRegion, sshUserName string) {
```

Comment:
`t.Parallel()`

Reply:
`t.Parallel()` is called higher up in [the tests](https://github.com/hashicorp/terraform-aws-vault/blob/master/test/vault_main_test.go#L80). Adding another causes a panic